OptimizelySDK/Odp/OdpEventApiManager.cs

```diff
@@ -89,7 +90,10 @@ public bool SendEvents(string apiKey, string apiHost, List<OdpEvent> events)
             }
 
             var endpoint = $"{apiHost}{Constants.ODP_EVENTS_API_ENDPOINT_PATH}";
-            var data = JsonConvert.SerializeObject(events);
+            var data = JsonConvert.SerializeObject(events, settings: new JsonSerializerSettings
+            {
+                ContractResolver = new CamelCasePropertyNamesContractResolver(),
```

Comment:
Looking through the ODP docs, it looks like they are using snake case for their field names. Maybe we should consider future proofing by utilizing a snake case strategy instead?